src/applications/coronavirus-vaccination/components/Confirmation.jsx

```diff
@@ -26,18 +26,74 @@ function Confirmation({ router, formData }) {
         vaccine options, we'll send you updates by email or text.
       </p>
       <p>
-        You can also visit our main{' '}
-        <a href="/health-care/covid-19-vaccine/">
-          COVID-19 vaccines at VA page
-        </a>{' '}
-        for the latest information and answers to common questions.
-      </p>
-      <p>
-        <strong>Remember:</strong> This form doesn’t sign you up to get a
-        vaccine. And you can change your mind about getting a vaccine at any
-        time. We’ll use the information you provided to understand your interest
-        and keep you informed. If you want to update your information later, you
-        can submit a new form.
+        Your local VA health facility may also use the information you provided
+        to determine when to contact you about getting a vaccine once your risk
+        group becomes eligible. If you told us you plan to get a vaccine, your
+        facility may contact you sooner. You can always change your mind about
+        getting a vaccine. And you can submit a new form to update your
+        information at any time.
+      </p>
+      <p>
+        <strong>Note for caregivers:</strong> If you’re a designated primary or
+        secondary caregiver in our{' '}
+        <a
+          href="/family-member-benefits/comprehensive-assistance-for-family-caregivers/"
+          target="_blank"
```

Comment:
Can we add `rel="noopener noreferrer"` and something to indicate to a screen reader that this link opens in a new window? Either `aria-label="Program of Comprehensive Assistance for Family Caregivers (Open in a new window)"` or some screen reader text?

Reference: https://timwright.org/blog/2017/01/21/opening-links/

Reply:
I don't think that this link _should_ open in a new window (unless that's a special req) bc typically links within www.va.gov open in the same tab